README.md

```diff
@@ -35,17 +35,21 @@ NB : System apps cannot really be uninstalled without root (see the [FAQ](https:
 * [ ] Gionee
 * [X] LG
 * [X] Google
+* [ ] iQOO
 * [X] Fairphone
 * [ ] HTC
 * [X] Huawei
 * [X] Motorola
 * [X] Nokia
 * [X] OnePlus
-* [X] Oppo  
+* [X] Oppo
+* [ ] Realme
```

Comment:
```suggestion
* [x] Realme
```

Realme is already supported with some packages, plus it uses a lot of the same apps as on OPPO phones. What do you think?